definitions/pipelines/rnaseq_star_fusion_with_xenosplit.cwl

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "STAR-RNA-Seq alignment and transcript/gene abundance workflow with Xenosplit"
+requirements:
+    - class: MultipleInputFeatureRequirement
+    - class: SubworkflowFeatureRequirement
+    - class: ScatterFeatureRequirement
+    - class: StepInputExpressionRequirement
+inputs:
+    instrument_data_bams:
+        type: File[]
+    outsam_attrrg_line:
+        type: string[]
+    graft_star_genome_dir:
+        type: Directory
+        doc: "Location of the STAR human reference directory"
+    graft_outfile_name_prefix:
+        type: string?
+        default: "Graft_"
+        doc: "Prefix name for the graft bam - human"
+    host_star_genome_dir:
+        type: Directory
+        doc: "Location of the STAR mouse reference directory"
+    host_outfile_name_prefix:
+        type: string?
+        default: "Host_"
+        doc: "Prefix name for the host bam - mouse"
+    star_fusion_genome_dir:
+        type: Directory
+        doc: "As the fusion would be called on only the final graftOutbam i.e. the human xenograft bam without any mouse reads we only need the star fusion directory for the human"
+    graft_gtf_file:
+        type: File
+        doc: "GTF file corresponding the to the Human STAR reference"
```

Comment:
Suggested change:
"GTF file corresponding the to the graft species (often human)"